include/queryfunctions.php

```diff
@@ -52,13 +54,17 @@ function GetWorkouts($userId){
             SELECT * FROM workouts
             WHERE isProgram IS NULL AND
             userId = ".$userId."
+            ORDER BY dateTimeStarted DESC
         ")->fetchAll();
         return $workouts; 
     }
 
-    function GetWorkout($workoutId,$userId){
-        $workout = GetWorkouts($userId);
-        return $workout[$workoutId];
+    function GetWorkout($workoutId){
+        $workout = dbQuery("
+            SELECT * FROM workouts
+            WHERE workoutId = ".$workoutId.""
```

Comment:
Online 42 of this file, you fixed a SQL injection issue by using `:userId`. You should do the same thing here. As a general rule: Never put a PHP variable in a database query directly.